### PR TITLE
Upgrade electron to 19.0.12

### PR DESCRIPTION
**Pull request by mamantoha**
This PR upgrades Electron to v19.0.12. Luckily, no API changes affected Mailspring :tada:

### CODE REVIEW

File: scripts/postinstall.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const npmEnvs = {
     npm_config_target: npmElectronTarget,
     npm_config_arch: process.platform === 'win32' ? 'ia32' : process.arch,
     npm_config_target_arch: process.platform === 'win32' ? 'ia32' : process.arch,
-    npm_config_disturl: 'https://atom.io/download/electron',
+    npm_config_disturl: 'https://electronjs.org/headers',
```
Comment (mamantoha) [inline scripts/postinstall.js]: Fixes `npm install`.

It fails with:

```
...
npm ERR! gyp http GET https://atom.io/download/electron/v19.0.10/node-v19.0.10-headers.tar.gz
npm ERR! gyp http 403 https://gh-contractor-zcbenz.s3.amazonaws.com/atom-shell/dist/v19.0.10/node-v19.0.10-headers.tar.gz
npm ERR! gyp WARN install got an error, rolling back install
...
```

Ref: https://www.electronjs.org/docs/latest/tutorial/using-native-node-modules#manually-building-for-electron

**Review by SpiritOTHawk — APPROVED** [review]

**Comment by ReillyBrogan** [comment]
19.0.13 now. There's also 20.0.x but it needs more to build than just bumping the electron version.

**Comment by ReillyBrogan** [comment]
Electron 19 is now end-of-life.

**Comment by Phylu** [comment]
Superseeded by https://github.com/Foundry376/Mailspring/pull/2444

**Pull request closed without merging**